pymongo/asynchronous/mongo_client.py

```diff
@@ -2204,10 +2224,136 @@ async def drop_database(
                 session=session,
             )
 
+    @_csot.apply
+    async def bulk_write(
+        self,
+        models: Sequence[_WriteOp[_DocumentType]],
+        session: Optional[AsyncClientSession] = None,
+        ordered: bool = True,
+        verbose_results: bool = False,
+        bypass_document_validation: Optional[bool] = None,
+        comment: Optional[Any] = None,
+        let: Optional[Mapping] = None,
+        write_concern: Optional[WriteConcern] = None,
+    ) -> ClientBulkWriteResult:
+        """Send a batch of write operations, potentially across multiple namespaces, to the server.
+
+        Requests are passed as a list of write operation instances (
+        :class:`~pymongo.operations.ClientInsertOne`,
+        :class:`~pymongo.operations.ClientUpdateOne`,
+        :class:`~pymongo.operations.ClientUpdateMany`,
+        :class:`~pymongo.operations.ClientReplaceOne`,
+        :class:`~pymongo.operations.ClientDeleteOne`, or
+        :class:`~pymongo.operations.ClientDeleteMany`).
+
+          >>> for doc in db.test.find({}):
```

Comment:
These inline examples should be asynchronous since this is in the asynchronous API.

Reply:
Is it okay for the inline examples in the synchronous API to be async? (since they get overwritten by the lint-manual)

Reply:
The inline examples should be converted to synchronous as part of the synchro process, are you seeing some remain async?

Reply:
When I add `async for` or `await` as part of the inline examples, they remain there in the sync version as well